src/contributing.md

```diff
@@ -322,6 +322,12 @@ Here are those same steps in detail:
 
 #### Updating submodules
 
+(As of <!-- date-check --> Sept 2022 `miri` is no longer a submodule but a subtree. The following
+instructions are still broadly correct for updating submodules in general. For a more specific
+example you can also see the steps for [upgrading llvm][upgrading-llvm].)
+
+[upgrading-llvm]: https://rustc-dev-guide.rust-lang.org/backend/updating-llvm.html
```

Comment:
We could just redirect to the "Git submodules" section on git.md and remove this to avoid duplication.

Reply:
I just removed that section altogether. We already link to the the "Git submodules" section above in the "External Dependencies (submodules)" section.